packages/flutter_tools/test/integration/expression_evaluation_test.dart

```diff
@@ -33,13 +33,13 @@ void main() {
 
     Future<VMIsolate> breakInBuildMethod(FlutterTestDriver flutter) async {
       return _flutter.breakAt(
-          Uri.file(_project.buildMethodBreakpointFile),
+          Uri.parse("package:test/main.dart"),
```

Comment:
I don't really understand what this change is doing (or why it's required). The breakpoint should be in the file defined in the test project, but this appears like it would break on an somewhat-random line in the test package?

Reply:
'package:test/main.dart' is the same thing as 'test/lib/main.dart', which causes problems if both are used in the app at the same time.
Frontend infers(and enforces) 'package:'-scheme uris for entry points (https://dart-review.googlesource.com/c/sdk/+/78246). So 'file:'-scheme uris are not recognized by vm.

Reply:
Oh sorry, I thought `package:test` was resolving to the test package. I guess this sample app doesn't reference that.

Is this a recent change? I'm not sure if Dart Code sends breakpoints for the "current project" as package-uris, so we might start missing breakpoints.

Also - minor nit, I think this should be fixed inside the Project classes to keep these in-sync. If buildMethodBreakpointFile is changed inside the project, this would fail. I don't mind picking that up tomorrow if you need to land this though (it LGTM, though be aware none of the checks are running right now due to GitHub issues).